packages/next/src/build/webpack/plugins/flight-client-entry-plugin.ts

```diff
@@ -460,7 +469,8 @@ export class FlightClientEntryPlugin {
         }
       }
 
-      return Promise.all(addedClientActionEntryList)
+      await Promise.all(addedClientActionEntryList)
+      return
     })
```

Comment:
Unnecessary change?

Reply:
This was changed related to typing updates, it needs to return `Promise<void>`. I tried tapAsync it will hang so I switched to a new line